website/sidebars.js

```diff
@@ -663,8 +663,8 @@ const sidebars = {
             },
             {
               type: 'doc',
-              id: 'run-arbitrum-node/arbos-releases/arbos31',
-              label: 'ArbOS 31 Bianca',
+              id: 'run-arbitrum-node/arbos-releases/arbos32',
```

Comment:
Can you create a redirection in `vercel.json` file? That way, if someone is still referencing the `arbos31` URL, the portal will redirect the user to `arbos32`.

Let me know if you need help creating the redirection 🙏

Reply:
thanks! done